lib/rucio/tests/test_meta.py

```diff
@@ -83,3 +88,63 @@ def xtest_add_value_to_bad_key(self):
     def xtest_add_key_mysql(self):
         """ META (CORE): Add a new key to test conversions on MySQL"""
         self.meta_client.add_key('datatype', DIDType.FILE)
+
+    def xtest_insert_metadata(self):
+        """ META (CLIENTS) : Adds a fully set json column to a did (rewrites if already present) """
+        data = {"key1": "value_" + str(uuid()), "key2": "value_" + str(uuid()), "key3": "value_" + str(uuid())}
+        self.meta_client.insert_metadata(scope=self.tmp_scope, name=self.tmp_name, meta=data)
```

Comment:
we generally use the convention: add, delete, list

so this function should be called ```add_metadata```